tests/nsrt_learning/test_nsrt_learning_main.py

```diff
@@ -201,6 +201,13 @@ def test_nsrt_learning_specific_nsrts():
                                   action_space,
                                   sampler_learner="random")
     assert len(nsrts) == 0
+    # Test minimum percent of examples parameter
+    utils.update_config({"min_perc_data_for_nsrt": 50})
```

Comment:
don't you also have to reset the `min_data_for_nsrt` parameter that was set in the above call to `update_config` since it's not `reset_config`

Reply:
and maybe explicitly set `min_perc_data_for_nsrt` to 0 up there?

Reply:
thanks good catch